testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/model/MapStoreProvider.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2022 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.testsuite.model;
+
+import org.keycloak.utils.StringUtil;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+/**
+ * @author <a href="mailto:mabartos@redhat.com">Martin Bartos</a>
+ */
+public enum MapStoreProvider {
+    CHM("chm") {
+        @Override
+        public Optional<String> getDbUrl() {
+            return Optional.of("");
+        }
+
+        @Override
+        public Optional<String> getDbUsername() {
+            return Optional.of("");
+        }
+
+        @Override
+        public Optional<String> getDbPassword() {
+            return Optional.of("");
+        }
+    },
+    JPA("jpa") {
+        @Override
+        public Optional<String> getDbUrl() {
+            return Optional.ofNullable(System.getProperty("keycloak.map.storage.connectionsJpa.url"));
+        }
+
+        @Override
+        public Optional<String> getDbUsername() {
+            return Optional.ofNullable(System.getProperty("keycloak.map.storage.connectionsJpa.user"));
+        }
+
+        @Override
+        public Optional<String> getDbPassword() {
+            return Optional.ofNullable(System.getProperty("keycloak.map.storage.connectionsJpa.password"));
+        }
+    },
+    HOTROD("hotrod") {
+        @Override
+        public Optional<String> getDbUrl() {
```

Comment:
@mabartos HotRod has different configuration options as DB in Quarkus dist, see https://github.com/keycloak/keycloak/blob/main/quarkus/tests/integration/src/test/resources/org/keycloak/it/cli/approvals/cli/help/HelpCommandTest.testStartDevHelpAll.unix.approved.txt#L82-L89

Reply:
@mhajas Thanks for your comment! I think the support for HotRod can be done in some follow-up task, WDYT? :)) 

Perhaps, included in this #13697 task?

Reply:
Sure we will make HotRod with Quarkus work with that issue. However, the changes done in this PR will need quite a significant refactoring to add the possibility of setting different options for HotRod. Maybe it would make sense to introduce a way of setting the option that already supports this. For example,  within the `MapStoreProvider` enum, we could have a method that takes `List<String> commands and it would do something like [this](https://github.com/keycloak/keycloak/pull/13643/files#diff-80935a251925bed2db7292d60c81a7546ceb14dcb4565e82cd5582e34705cc4cL250-L263) but each case would be a different enum. WDYT? If you think it makes sense to solve this later and this PR needs to be merged asap, we can leave this as is.

Reply:
Yep, it makes sense, at least, to prepare the codebase for the HotRod changes :)) I'll do it in this PR